api/routes/reviewRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const { reviewController } = require("../controllers");
+const { loginRequired } = require("../utils/auth");
+
+const router = express.Router();
+
+router.post("/create", reviewController.createReview);
+router.post("/read", reviewController.getReview);
+router.patch("/update", reviewController.updateReview);
+router.delete("/delete", reviewController.deleteReview);
```

Comment:
REST를 이해했다면 end point에는 동사가 작성될 필요가 없습니다. REST 아키텍쳐 기반으로 API 개발시 이러한 엔드포인트를 설계할 때에는 가장 단순하게 What, How 두가지만 고려하면 되는데 what은 uri 리소스를 통해 목적 자원을 명시할 수 있고, http request method를 통해서 액션을 명시할 수 있습니다.
```suggestion
router.post("", reviewController.createReview);
router.get("/:movieId", reviewController.getReview);
router.patch("", reviewController.updateReview);
router.delete("", reviewController.deleteReview);
```